src/CommunityToolkit.Maui.Markup/AbsoluteLayoutExtensions.cs

```diff
@@ -0,0 +1,120 @@
+﻿using Microsoft.Maui.Controls;
```

Comment:
I understand the complexity in naming this class but `AbsoluteLayoutExtensions` makes me think that they are extension methods that apply to `AbsoluteLayout`, I get that they related to it though.

Would something like `AbsoluteLayoutChildExtensions` make sense? I am not sure it does re-reading it...

Reply:
I am also not entirely sure it really matters

Reply:
>. AbsoluteLayoutExtensions makes me think that they are extension methods that apply to AbsoluteLayout 

Yea, that's totally fair. Once could argue that you wouldn't use these extension methods unless you were using an `AbsoluteLayout`.

> I am also not entirely sure it really matters

Haha - I totally agree! Since they're all extension methods, the class name isn't visible to any consumers of the `CommunityToolkit.Maui.Markup` NuGet package.